app/javascript/react/containers/ParkShowContainer.js

```diff
@@ -7,9 +7,24 @@ class ParkShowContainer extends Component {
     super(props);
     this.state ={
       park: {
-        reviews: []
+        reviews: [],
+        selectedThumbsUpId: null,
+        selectedThumbsDownId: null
       }
     }
+    this.handleReviewThumbsUpClick = this.handleReviewThumbsUpClick.bind(this)
+    this.handleReviewThumbsDownClick = this.handleReviewThumbsDownClick.bind(this)
+  }
+
+  handleReviewThumbsUpClick( id ) {
```

Comment:
Watch your whitespace! There should be an empty line between these two methods, no space on either side of `id` here, and no space between `handleReviewThumbsDownClick` and `(id)` below!